src/main/java/service/FileService.java

```diff
@@ -0,0 +1,10 @@
+package service;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface FileService {
+    List<String> readFromFile(String filePath) throws IOException;
+
+    void writeToFile(String filePath, String report) throws IOException;
```

Comment:
Почему имя переменной **String report**?
```suggestion
    List<String> readFromFile(String filePath);

    void writeToFile(String filePath, String report);
```